src/components/Panel/PanelContainer.js

```diff
@@ -142,18 +152,48 @@ export default class PanelContainer extends Component {
       secondaryButton,
       subtitle,
       title,
+      hasScrollingContent,
+
+      // Explicitly define so they aren't in `...other`.
+      disableEscape,
```

Comment:
I wasn't able to find where this is or isn't being used - can you confirm that we need this?